pkg/dockerregistry/server/auth.go

```diff
@@ -295,10 +320,19 @@ func (ac *AccessController) Authorized(ctx context.Context, accessRecords ...reg
 	}
 
 	// In case of docker login, hits endpoint /v2
-	if len(accessRecords) == 0 {
-		if err := verifyOpenShiftUser(ctx, osClient); err != nil {
+	if len(bearerToken) > 0 {
+		if user, userid, err := verifyOpenShiftUser(ctx, osClient); err != nil {
 			return nil, ac.wrapErr(ctx, err)
+		} else {
+			ctx = WithUserInfoLogger(ctx, user, userid)
 		}
+	} else {
+		ctx = WithUserInfoLogger(ctx, defaultUserName, "")
```

Comment:
@miminar No. I want to get user information for every `bearerToken` regardless of there access records or not. Even if the request does not require permissions, I want to see his name.